src/components/Blogs/index.jsx

```diff
@@ -3,24 +3,38 @@ import { Container, Row } from "react-bootstrap";
 import SearchBar from "../SearchBar";
 import BlogCard from "../BlogCard";
 import "./index.scss";
+import i18next from "i18next";
 
-const POSTS_PER_PAGE = 9;
-const BLOG_API_URL = `https://public-api.wordpress.com/wp/v2/sites/safeplace102505649.wordpress.com/posts?per_page=${POSTS_PER_PAGE}`;
+//  "10221" and "38299" are reference numbers from wordpress that we get after tagging each post to its language in order to make the change between the languages
+const EN_REFERENCE_NUMBER = "10221";
+const AR_REFERENCE_NUMBER = "38299";
 
 export default function Blogs() {
   const [blogPosts, setBlogPosts] = useState([]);
   const [blogsForSearch, setBlogsForSearch] = useState([]);
+  const [languageRef, setLanguageRef] = useState(EN_REFERENCE_NUMBER);
 
-  const fetchBlogPosts = async () => {
-    const data = await fetch(BLOG_API_URL);
-    const fetchedBlogPosts = await data.json();
-    setBlogsForSearch(fetchedBlogPosts);
-    setBlogPosts(fetchedBlogPosts);
-  };
+  i18next.on("languageChanged", (lng) => {
+    let ref = i18next.translator.language;
+    if (ref === "en") {
+      setLanguageRef(EN_REFERENCE_NUMBER);
+    } else if (ref === "ar") {
+      setLanguageRef(AR_REFERENCE_NUMBER);
+    }
+  });
 
   useEffect(() => {
+    const blogsApiUrl = `https://public-api.wordpress.com/wp/v2/sites/safeplace102505649.wordpress.com/posts?tags=${languageRef}`;
```

Comment:
move the prefix back to BLOG_API_URL, add ${BLOG_API_URL}?tags=${languageRef}

Reply:
${BLOG_API_URL}?tags=${languageRef} this should be in a separate variable?